CryptoPkg/CryptoPkg.dec

```diff
@@ -20,9 +20,7 @@
   Include
 
 [Includes.Common.Private]
-  Private
   Library/Include

```

Comment:
is this valid?  Does this folder exist and if so what module uses it?

Reply:
The Private folder was used for the crypto binary protocols and PPI as well as some library header files for IntrinsicLib and OpensslLib.  All of these were either removed or moved to be public so I removed the reference to the folder.

Reply:
I misunderstood.  If you're referencing the Library/Include then you're right.  I'll make an update to fix that.

Reply:
They've been removed as well.